common/core/desktop/tests/unit/kmx/049 - groups ctrlalt.kmn

```diff
@@ -0,0 +1,28 @@
+c Description: Tests multiple groups. Purpose here is to interleave output and backspaces from kmx processor to consumer
```

Comment:
Looks like this test doesn't run (because it's not listed in `meson.build`). I see that it's used in `kmx_key_list.cpp`. If it's intended to be used only from that test than IMO it would be better and less confusing to rename it to `kmx_key_list.kmn` or something like that.

Reply:
Agreed

Reply:
Yes, when I was first setting it up I as going to create a new `expected key list` comment to go in the header of the kmn so that it would test with `kmx.cpp`. I ended up with having it all testested from `kmx_key_list.cpp` I will rename this to the suggested name.

Reply:
@ermshiperete  I fixed the what I thought where the issues with kmx_key_list and the meson.build, but the jenkins build is still failing. Can you also add my user to the build so I can view it in Jenkins?